parseComCode/main.js

```diff
@@ -30,6 +30,61 @@ Parse.Cloud.job("populateSearch", function(request, status) {
   });
 });
 
+// This job will remove any language records which currently do not have any books which use them.
+// The purpose is to keep BloomLibrary.org from displaying languages with no books.
+// It is scheduled to run every day.
```

Comment:
I think it would be useful to comment here on what makes that so (where you would go to adjust the frequency, turn it off, etc.)